service/javadsl/integration-tests/src/test/java/com/lightbend/lagom/it/mocks/AdditionalRoutersService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.it.mocks;
+
+import com.lightbend.lagom.javadsl.api.Descriptor;
+import com.lightbend.lagom.javadsl.api.Service;
+
+import akka.NotUsed;
+import com.lightbend.lagom.javadsl.api.*;
+import static com.lightbend.lagom.javadsl.api.Service.*;
+
+public interface AdditionalRoutersService extends Service {
+
+    @Override
+    default Descriptor descriptor() {
+        return named("additional-routers")
+
+            .withServiceAcls(
+                ServiceAcl.path("/ping"),
+                ServiceAcl.path("/pong")
```

Comment:
Do the `hello` and `hello-prefixed` routes still also have to be added here in order to work?

Reply:
oh, good point. I forget about them. I have no idea how this is working. :-(

Reply:
@raboof this is not needed because of the way we are running the tests. The ACLs are only needed in case we want to expose the app to a web gateway, which is the case of Lagom's dev mode for instance. 

In this test, we are starting the app and using Play's test Helpers to fabricate requests and sent it directly to the app.

Reply:
Right,  but maybe good to add them anyway to make the test a bit more representative of how this API would have to be used by users?